### PR TITLE
Adding documentation on hosted pages

### PR DESCRIPTION
**Pull request by jeffreylees**

### CODE REVIEW

**Comment by auth0build** [comment]
Can one of the admins verify this patch?

**Comment by ntotten** [comment]
Make sure to update the sidebar.yml

File: articles/hosted-pages/login.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+<img width="500" src="/media/articles/hosted-pages/login.png" />
+
+If you want to change some of the [configuration options](/libraries/lock/v10/customization) within Lock, you may do so _right on this page_, just make your changes and make sure to remember to hit the "save" button. 
+
+This login page will be a basic login page for your client, and will use Lock to provide your users with a beautiful and smooth authentication process. The hosted login page is both one of your most secure authentication options as well as one of the easiest to implement.
+
+You will be able to access your hosted login page via the following url:
+
+```text
+https://'${account.namespace}'/login?client='${account.clientId}'
```
Comment (ntotten) [inline articles/hosted-pages/login.md]: Extra quotes around `'${account.namespace}'`

File: articles/hosted-pages/password-reset.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+
+# Password Reset Page
+
+In your [Auth0 Dashboard](https://manage.auth0.com/#/password_reset), you can enable your Hosted Password Reset Page by simply flipping the toggle switch.
+
+<img width="500" src="/media/articles/hosted-pages/password_reset.png" />
```
Comment (ntotten) [inline articles/hosted-pages/password-reset.md]: Shouldn't set widths. Use markdown ``

File: articles/hosted-pages/error-pages.md [hunk 3]
```diff
@@ -1,12 +1,10 @@
----
-description: How use Auth0's built-in generic error pages.
----
+# Error Pages
```
Comment (mpaktiti) [inline articles/hosted-pages/error-pages.md]: Can you add description metadata for the doc please?

File: articles/hosted-pages/guardian.md [hunk 4]
```diff
@@ -0,0 +1,11 @@
+
+# Guardian Multifactor Login Page
```
Comment (mpaktiti) [inline articles/hosted-pages/guardian.md:5]: Can you add description metadata for the doc please?

File: articles/hosted-pages/guardian.md [hunk 5]
```diff
@@ -0,0 +1,11 @@
...
+
+In your [Auth0 Dashboard](https://manage.auth0.com/#/guardian_mfa_page), you can enable your Hosted Guardian Multifactor Login Page by simply flipping the toggle switch.
```
Comment (mpaktiti) [inline articles/hosted-pages/guardian.md]: Hardcoded value for the dashboard. Use the manage_url variable instead.

File: articles/hosted-pages/guardian.md [hunk 6]
```diff
@@ -0,0 +1,11 @@
...
+
+![](/media/articles/hosted-pages/guardian.png)
```
Comment (mpaktiti) [inline articles/hosted-pages/guardian.md]: Missing alternate text for the image.

File: articles/hosted-pages/guardian.md [hunk 7]
```diff
@@ -0,0 +1,11 @@
...
+
+If you want to change some of the options in the Change Password Widget, you may do so _right on this page_, just make your changes and make sure to remember to hit the "save" button. The options have comments to help document their uses.
```
Comment (mpaktiti) [inline articles/hosted-pages/guardian.md]: Replace `"save"` with `__Save__`.

File: articles/hosted-pages/index.md [hunk 8]
```diff
@@ -0,0 +1,47 @@
...
+
+Auth0 provides you the ability to create beautiful hosted pages to which you can redirect to provide functionality for your users. These pages include [Login](/hosted-pages/login), [Password Reset](/hosted-pages/password-reset), [Guardian Multifactor](/hosted-pages/guardian), and [Error pages](/hosted-pages/error-pages).
+
+## Security Considerations
+ 
+Using Auth0 hosted pages for your authentication, rather than externally hosting them, provides seamless XSRF protection, preventing third party impersonation or hijacking of sessions. The use of hosted pages provides a significantly easier to implement, secure solution for authentication, allowing you to worry about other things, like the security of your application.
+
+## How to Customize Hosted Pages
+
+You can customize your hosted pages from within your [Auth0 Dashboard](https://manage.auth0.com). See the following pages for specific instructions about each type of hosted page:
```
Comment (mpaktiti) [inline articles/hosted-pages/index.md]: Replace hardcoded dashboard URL with variable

File: articles/hosted-pages/login.md [hunk 9]
```diff
@@ -0,0 +1,19 @@
+
+# Login Page
+
+In your [Auth0 Dashboard](https://manage.auth0.com/#/login_page), you can enable your Hosted Login Page by simply flipping the toggle switch.
```
Comment (mpaktiti) [inline articles/hosted-pages/login.md]: Replace hardcoded dashboard URL with variable

File: articles/hosted-pages/login.md [hunk 10]
```diff
@@ -0,0 +1,19 @@
...
+
+![](/media/articles/hosted-pages/login.png)
```
Comment (mpaktiti) [inline articles/hosted-pages/login.md]: Missing alternate text for the image.

File: articles/hosted-pages/login.md [hunk 11]
```diff
@@ -0,0 +1,19 @@
...
+
+If you want to change some of the [configuration options](/libraries/lock/v10/customization) within Lock, you may do so _right on this page_, just make your changes and make sure to remember to hit the "save" button.
```
Comment (mpaktiti) [inline articles/hosted-pages/login.md]: Replace `"save"` with `__save__`

File: articles/hosted-pages/password-reset.md [hunk 12]
```diff
@@ -0,0 +1,11 @@
+
+# Password Reset Page
+
+In your [Auth0 Dashboard](https://manage.auth0.com/#/password_reset), you can enable your Hosted Password Reset Page by simply flipping the toggle switch.
+
+![](/media/articles/hosted-pages/password_reset.png)
```
Comment (mpaktiti) [inline articles/hosted-pages/password-reset.md]: Missing alternate text for the image.

File: articles/hosted-pages/error-pages.md [hunk 13]
```diff
@@ -1,12 +1,10 @@
...
 
-# Auth0's Generic Error Pages
+## Generic Error Page
 
-Throughout the authentication process, your users may encounter errors. Auth0 provides you the option of using [custom error pages](/custom-error-pages), but you may also choose instead to use the generic error page that Auth0 provides to alert the user of said errors.
+Throughout the authentication process, your users may encounter errors. Auth0 provides you the option of using [custom error pages](/hosted-pages/custom-error-pages), but you may also choose instead to use the generic error page that Auth0 provides to alert the user of said errors.
 
-![](/media/articles/error-pages/generic-error-page-settings.png)
+![](/media/articles/hosted-pages/generic-error-page-settings.png)
```
Comment (mpaktiti) [inline articles/hosted-pages/error-pages.md]: Missing alternate text for the image.

**Review by mpaktiti — CHANGES_REQUESTED** [review]
Hey @jeffreylees the doc needs some edits, please see my comments. Also, all three screenshots are larger that the specs (1260 × 1053), please downsize to 900px wide. Thanks!

**Pull request merged**